### ISSUE TITLE
How to delete a stored password?

### ISSUE THREAD

**Issue by hsyhhssyy** [intersystems-community/intersystems-servermanager#192]
If I incorrectly entered a password when logging into a server for the first time and pressed the Key button, a wrong password would be saved somewhere unknown, causing an issue where I can't use the saved password for future logins and have to manually enter it each time. 

How can I modify or delete the saved password? I have looked through all the documentation but couldn't find any instructions on this matter.

**Comment by gjsjohnmurray** [comment]
Does the signing out info [in the README ](https://github.com/intersystems-community/intersystems-servermanager#managing-signed-in-accounts)  help?

**Comment by hsyhhssyy** [comment]
This solution has partially solved my issue, as indeed I can log out and be prompted to delete the password here. However, there's another problem.

![image](https://github.com/intersystems-community/intersystems-servermanager/assets/5977110/2ec54fe4-dfb5-4ab9-bfc4-efed46c15c2f)

After clicking the link once, it not only asks whether to delete the password for the currently connected server but also for every account with the same name. I have more than 20 servers saved in the system, all with the same username, hence this dialogue box keeps popping up over 20 times. I need to carefully read the content of each pop-up to make sure I'm deleting the correct password.

Regardless, at least I now have a way to delete the stored passwords. Thank you for your help.

**Comment by gjsjohnmurray** [comment]
@hsyhhssyy I have created a new issue to track investigation of that.

@isc-bsaviano please liaise with InterSystems documentation team to get [the information about the authentication provider](https://github.com/intersystems-community/intersystems-servermanager#the-authentication-provider) rolled into [the new docs](https://docs.intersystems.com/components/csp/docbook/DocBook.UI.Page.cls?KEY=GVSCO).

**Comment by hsyhhssyy** [comment]
@gjsjohnmurray 
Thank you very much for your response. I have installed this plugin, and after installation, credentials for the same username but different servers are now listed separately, as shown in the image below.

![image](https://github.com/intersystems-community/intersystems-servermanager/assets/5977110/34842f14-c506-48b8-a76f-f9494d35da40)

When I tried to log out of one of them, the prompt asking if I want to delete the password only includes the server I selected. So far, I have not discovered any other issues.
